drivers/filesystems/reiserfs/src/read.c

```diff
@@ -196,7 +196,7 @@ RfsdReadVolume (IN PRFSD_IRP_CONTEXT IrpContext)
                 }
 
                 BlockArray.Irp = NULL;
-                BlockArray.Lba = ByteOffset.QuadPart;;
+                BlockArray.Lba = ByteOffset.QuadPart;
```

Comment:
3rd party code?

Reply:
Do you want me to revert this one ? Or do we take it as it won't harm ?
I only took out code in 3rdparty folder

Reply:
Revert.